src/components/content/common/deployedServiceDetailsByServiceIdQuery/useServiceDetailsByServiceIdQuery.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { useQuery } from '@tanstack/react-query';
+import {
+    getSelfHostedServiceDetailsById,
+    GetSelfHostedServiceDetailsByIdData,
+    getVendorHostedServiceDetailsById,
+    GetVendorHostedServiceDetailsByIdData,
+    serviceHostingType,
+    taskStatus,
+} from '../../../../xpanse-api/generated';
+
+export function useServiceDetailsByServiceIdQuery(
```

Comment:
the same query there here as well - https://github.com/eclipse-xpanse/xpanse-ui/blob/accd7dde31c42d75a837e278ae8366e99419b457/src/components/content/order/orderStatus/useServiceDetailsPollingQuery.ts#L51

please consolidate.

Reply:
consolidated.